src/assets/scripts/service-vtunnel-peer.initd

```diff
@@ -0,0 +1,16 @@
+#!/sbin/openrc-run
+
+# This script is used to manage vtunnel's peer process via OpenRC.
+
+# shellcheck shell=ksh
+
+name="vtunnel-peer"
+description="Rancher Desktop peer process for vtunnel"
```

Comment:
This should say it's vtunnel for credhelper — we'll want extra vtunnels for other processes (at least in the current architecture).

This also implies renaming the file.